### PR TITLE
Various Powershell Fixes

### PR DESCRIPTION
**Pull request by Shammah**
**What type of PR is this?**

Bug fix

**What does this PR do? Why is it needed?**

This PR fixes various trivial Powershell script bugs, mostly related to the output of the usage description.

**Which issues(s) does this PR fix?**

None

**Other notes for review**

Because of the old and outdated nature of `.bat` files, I suggest updating the documentation to advice the usage of the `.ps1` script instead of the `.bat` file. You could always suggest to use it is a backup in case the user is working on an old PC.

### CODE REVIEW

File: prysm.ps1 [hunk 1]
```diff
@@ -14,12 +13,12 @@ Use this script to download the latest Prysm release binaries.
 Downloaded binaries are saved to .\dist
  
 To specify a specific release version:
-  $env:USE_PRYSM_VERSION=v1.0.0-alpha3
+  `$env:USE_PRYSM_VERSION="v1.0.0-beta.3"
```
Comment (prestonvanloon) [inline prysm.ps1:16]: There is only one back tick here, do you need a closing back tick?

Comment (Shammah) [reply prysm.ps1:16]: The escape character for Powershell is a backtick `, where it for example is `\" in some other programming languages. No closing back tick is necessary.

**Review by prestonvanloon — APPROVED** [review]

File: prysm.ps1 [hunk 2]
```diff
@@ -3,7 +3,6 @@ $ProgressPreference = 'SilentlyContinue' # Disable Invoke-WebRequest progress ba
 
 # Complain if invalid arguments were provided.
 if ("beacon-chain", "validator", "slasher" -notcontains $args[0]) {
-    Write-Host "ERROR: prysm is only supported on 64-bit Operating Systems" -ForegroundColor Red;
```
Comment (prestonvanloon) [inline prysm.ps1:6]: Any reason to remove this?

Comment (Shammah) [reply prysm.ps1:6]: The error should only show up if you don't have a 64 bit system, which is what is being checked below this piece of code at line 30. Showing this "error" in full red confuses the user if he's already on a 64 bit system, which makes him or her think there's something wrong when there isn't and this message can safely be ignored.

**Pull request merged**